CHANGES.md

```diff
@@ -1,6 +1,43 @@
 
 # Changes
 
+## edge-21.1.3
+
+This edge release improves proxy diagnostics and handling for the failfast state
```

Comment:
Probably most readers are unfamiliar with what failfast means.  What about something like: "... improves proxy diagnostics and recovery in situations where the proxy is temporarily unable to route requests". ? @olix0r wdyt?

Reply:
+1